doc/index.rst

```diff
@@ -0,0 +1,7 @@
+.. _ros2_control_demos:
+
+#################
+Demos
```

Comment:
Which infos should we move from README.md to this rst file? All except build instructions and build?

If someone starts from control.ros.org with installed debian packages, the `Getting started` section is maybe not necessary in the documentation but only in the readme if the repo is checked out?

What about the figures in the doc folder, they are at least not up-to-date?

Reply:
I made a suggestion, feel free to comment.
I had to rename the `README.rst` to `doc/userdoc.rst,` because it would be [excluded otherwise](https://github.com/ros-controls/control.ros.org/blob/master/conf.py#L71). Is there a reason for that?

Reply:
Also I think rst would be upset about the too many ###?

Reply:
sphinx did not complain. I think the type of character does not matter.